uiowa/uiowa
# update stage_file_proxy to 3.1.3

# How to test

- Check that your local.settings.php file for the website is not trying to mimic another environment like PROD. `
$config['config_split.config_split.local']['status'] = FALSE;` (We want that to be TRUE if it exists at all and not overridden by another line like `config_split.config_split.prod`
- Sync sandbox.uiowa.edu but delete the contents of the site's files directory locally.
- If you haven't previously, update the local.settings.php file for the website with the internal production or live URL of the website (e.g. `$config['stage_file_proxy.settings']['origin'] = 'https://sandbox.prod.drupal.uiowa.edu/';`)
- Database updates should run without error and config should import locally as expected.
- Navigate pages and see media display
- If hot linking is enabled, media should display and the files directory should remain fairly empty.
- If hot linking is disabled, media should be downloaded to the files directory as expected.

admin/config/system/stage_file_proxy
